### ISSUE TITLE
SonarKit podspec validation fails

### ISSUE THREAD

**Issue by priteshrnandgaonkar** [facebook/flipper#11]
Currently `pod spec lint SonarKit.podspec` fails and its blocking us to release it on cocoapods. It fails the following error

```shell
-> SonarKit (0.0.1)
    - WARN  | source: Git sources should specify a tag.
    - WARN  | url: The URL (https://github.com/facebook/Sonar) is not reachable.
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] swift: The validator used Swift 3.2 by default because no Swift version was specified. To specify a Swift version during validation, add the `swift_version` attribute in your podspec. Note that usage of the `--swift-version` parameter or a `.swift-version` file is now deprecated.
    - ERROR | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | [SonarKit/SonarKitLayoutComponentKitSupport] xcodebuild:  warning: no rule to process file 'ComponentKit/ComponentKit/ComponentKit.xcconfig' of type text.xcconfig for architecture i386
    - NOTE  | [SonarKit/SonarKitLayoutComponentKitSupport] xcodebuild:  warning: no rule to process file 'ComponentKit/ComponentKit/ComponentKit.xcconfig' of type text.xcconfig for architecture x86_64
    - WARN  | [SonarKit/SonarKitLayoutPlugin,SonarKit/SonarKitNetworkPlugin] xcodebuild:  glog/src/demangle.cc:343:18: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'int' [-Wshorten-64-to-32]
    - WARN  | [SonarKit/SonarKitLayoutPlugin,SonarKit/SonarKitLayoutComponentKitSupport] xcodebuild:  DoubleConversion/double-conversion/strtod.cc:209:7: warning: code will never be executed [-Wunreachable-code]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:70:81: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - NOTE  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.2.sdk/usr/include/dispatch/queue.h:343:1: note: 'dispatch_get_current_queue' has been explicitly marked deprecated here
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:90:68: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:200:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:204:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:205:8: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:207:5: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:265:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:266:35: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:267:8: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:268:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:345:53: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:351:5: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:353:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:354:8: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:354:54: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:355:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:395:37: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:397:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:462:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:477:33: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:478:22: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:479:19: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:483:23: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:484:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:493:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:499:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:504:15: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:506:14: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTProtocol.m:79:38: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTProtocol.m:314:69: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:217:29: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:324:28: warning: 'dispatch_get_current_queue' is deprecated: first deprecated in iOS 6.0 - unsupported interface [-Wdeprecated-declarations]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:416:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:416:26: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:417:6: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:441:16: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:444:57: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:446:10: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:454:9: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:472:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:492:22: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:492:64: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:498:7: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:492:54: warning: implicit conversion loses integer precision: 'long long' to 'size_t' (aka 'unsigned long') [-Wshorten-64-to-32]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:567:62: warning: implicit conversion loses integer precision: 'unsigned long' to 'socklen_t' (aka 'unsigned int') [-Wshorten-64-to-32]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTChannel.m:589:60: warning: values of type 'NSInteger' should not be used as format arguments; add an explicit cast to 'long' instead [-Wformat]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:88:49: warning: implicit conversion loses integer precision: 'NSUInteger' (aka 'unsigned long') to 'uint32_t' (aka 'unsigned int') [-Wshorten-64-to-32]
    - WARN  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  PeerTalk/peertalk/PTUSBHub.m:385:33: warning: implicit conversion loses integer precision: 'NSInteger' (aka 'long') to 'USBMuxReplyCode' (aka 'unsigned int') [-Wshorten-64-to-32]
    - NOTE  | [SonarKit/SonarKitLayoutPlugin, SonarKit/SonarKitLayoutComponentKitSupport, SonarKit/SonarKitNetworkPlugin, and more...] xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:135:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:137:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:141:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:143:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:170:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:624:33: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:757:65: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:782:33: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:792:33: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1073:47: warning: implicit conversion loses integer precision: 'size_type' (aka 'unsigned long') to 'const int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1334:19: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'const int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1406:40: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'const int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1507:67: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1520:67: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1887:21: warning: implicit conversion loses integer precision: 'ssize_t' (aka 'long') to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/logging.cc:1888:16: warning: implicit conversion loses integer precision: 'ssize_t' (aka 'long') to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/signalhandler.cc:95:50: warning: implicit conversion loses integer precision: 'long' to 'int' [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/vlog_is_on.cc:52:1: warning: implicit conversion loses integer precision: 'long' to 'google::int32' (aka 'int') [-Wshorten-64-to-32]
    - WARN  | [iOS] [SonarKit/SonarKitLayoutPlugin] xcodebuild:  glog/src/vlog_is_on.cc:163:27: warning: implicit conversion loses integer precision: 'size_t' (aka 'unsigned long') to 'const int' [-Wshorten-64-to-32]
```

**Comment by colinhumber** [comment]
By default, pushing to a spec repo fails if there are warnings emitted. If you want to override this behavior, you can push to the spec repo by including the `--allow-warnings` argument. Otherwise, the warnings would need to be fixed.

**Comment by colinhumber** [comment]
The Swift version should be specified in the podspec as outlined here. https://guides.cocoapods.org/syntax/podspec.html

**Comment by priteshrnandgaonkar** [comment]
I tried with `swift_version` but it didn't validate the pod. `--allow-warnings` will just get rid off the warnings but there is an error while validating podspec.

<img width="1573" alt="screen shot 2018-06-12 at 11 40 16 am" src="https://user-images.githubusercontent.com/3865908/41285851-71b06126-6e35-11e8-82ac-2f1f33f4a9da.png">

**Comment by minikin** [comment]
@priteshrnandgaonkar  Have you tried just to build Xcode project for the pod? 
I've just tried and can't build it.

**Comment by noahsark769** [comment]
@priteshrnandgaonkar did this get fixed? If not, happy to look into it myself as well

**Comment by priteshrnandgaonkar** [comment]
It didn't get fixed yet? @noahsark769, you can take a look at it. We would need this to be fixed so that we can push it to cocoapods

**Comment by pkrmf** [comment]
1.- First of all SonarKit is pulling PeerTalk 0.0.1 as a dependency. PeerTalk 0.0.1 is broken since it is not defining the public constants in the header files as `extern const` or using `FOUNDATION_EXPORT`. That is something the owner of PeerTalk fixed in release 0.0.2, but he updated his podspec to point to the tag 0.0.2 which doesn't even exist on his [repo](https://github.com/rsms/peertalk). So 2 things need to happen regarding PeerTalk: Podspec needs to be updated and pushed to the cocoapods master repo. 
2.- The actual podspec is completely broken. It only generates 2 of the 4 subspecs, it actually doesn't even pull any of the default sources of SonarKit.


If I have some time tomorrow I can take a look at it and try to fix it

**Comment by noahsark769** [comment]
Since the podspec now validates using the cocoapods private repo, I think we can close this issue in favor of #132
